.github/workflows/rollout.yml

```
@@ -250,3 +250,31 @@ jobs:
       ONEPASSWORD_VAULT: ${{ secrets.ONEPASSWORD_VAULT }}
       ETHERPAD_API_KEY: ${{ secrets.ETHERPAD_API_KEY }}
       KUBECONFIG: ${{ secrets.KUBECONFIG }}
+
+  infra-schulcloud-01:
+    uses: hpi-schul-cloud/dof_app_deploy/.github/workflows/host.yml@main
+    with:
+      cfg_version: ${{ github.event.inputs.config_tag_name }}
+      host_name: infra.schulcloud-01
+      host_group: default
+      host_stage: infra
+      execute_group: ${{ github.event.inputs.instance_group }}
+    secrets:
+      token: ${{ secrets.GITHUB_TOKEN }}
+      ONEPASSWORD_VAULT: ${{ secrets.ONEPASSWORD_VAULT }}
+      ETHERPAD_API_KEY: ${{ secrets.ETHERPAD_API_KEY }}
+      KUBECONFIG: ${{ secrets.KUBECONFIG }}
+
+  infra-schulcloud-02:
```

please put it behind the Approval stage approve_ref
```suggestion
  infra-schulcloud-02:
    needs:
      - approve_ref 
```

---

The infra-schulcloud instances are for testing infrastructure and run on dedicated infra-schulcloud Kubernetes clusters, they don't need approval